docs/modules/game-object-creation/custom-usage.md

```diff
@@ -0,0 +1,52 @@
+# Custom spawning logic
+
+The Game Object Creation Feature Module offers you a simple way to customize how GameObjects are created. You must:
+
+1. Implement a class which inherits from the `IEntityGameObjectCreator` interface.
+2. Tell the GDK to use this implementation.
+
+## Implementing the `IEntityGameObjectCreator` interface
```

Comment:
nit: you use different verb forms (e.g. here "Implementing", somewhere else "Add"). We should standardize it where possible

Reply:
Agreed